tiledb/sm/misc/stats_counters.h

```diff
@@ -129,19 +129,20 @@ STATS_DEFINE_FUNC_STAT(vfs_is_dir)
 STATS_DEFINE_FUNC_STAT(vfs_is_empty_bucket)
 STATS_DEFINE_FUNC_STAT(vfs_is_file)
 STATS_DEFINE_FUNC_STAT(vfs_ls)
-STATS_DEFINE_FUNC_STAT(vfs_move_file)
 STATS_DEFINE_FUNC_STAT(vfs_move_dir)
+STATS_DEFINE_FUNC_STAT(vfs_move_file)
```

Comment:
You also have to add the INIT and REPORT lines if you want to add the new stats counter.